tailwind.config.js

```diff
@@ -2,7 +2,14 @@
 module.exports = {
   content: ["./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        podCyan: "rgb(var(--pod-cyan) / <alpha-value>)",
```

Comment:
Oh, I see. Can we only use the values here rather than having it as a global CSS value?